### ISSUE TITLE
Compliance/Compliance Summary workflow - Apply Compliance Units button

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-compliance#96]
#### Description:

As an external user, I can click the Apply Compliance Units button on the Compliance Summary page (Compliance Report 2)

[Design in Figma](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=32714-10485&t=TL1kcoSNpFI0Dxy9-4)

#### Acceptance Criteria:

Given that I am an external (industry) user
When navigate to the Review Compliance Summary page
Then I can click the Apply Compliance Units button and open the next page [Apply Compliance Units](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=32707-6934&t=TL1kcoSNpFI0Dxy9-4)

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

#### Definition of Ready (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

#### Definition of Done (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By/Blocking

Blocking #145 and #147 
#### Notes:

-

**Comment by fviduya** [comment]
I have tested this alongside #133, I can verify that the `Apply Compliance Units` button is visible in the Compliance Summary page. The workflow itself will be under #145. Moving to Pending PO for review of test. Thank you!

**Comment by vesselak** [comment]
Reviewed the QA documentation and the button is visible in dev. Accepting ticket and will update #145 with the end point ticket link.
